srtcore/buffer.cpp

```diff
@@ -582,45 +612,24 @@ int CSndBuffer::getAvgBufSize(int& w_bytes, int& w_tsp)
     /* update stats in case there was no add/ack activity lately */
     updAvgBufSize(steady_clock::now());
 
-    w_bytes = m_iBytesCountMAvg;
-    w_tsp   = m_TimespanMAvg;
-    return (m_iCountMAvg);
+    // Average number of packets and timespan could be small,
+    // so rounding is beneficial, while for the number of
+    // bytes in the buffer is a higher value, so rounding can be omitted,
+    // but probably better to round all three values.
+    w_bytes = static_cast<int>(round(m_mavg.bytes()));
```

Comment:
How about having `round_bytes()` etc. functions that would use `round` inside and return already the expected integer type? That would separate better the application code and all calculations.

Reply:
Maybe `round_value()' then? Because it is not only bytes, but also packets and timespan.